opennms-container/sentinel/Dockerfile

```diff
@@ -38,40 +53,37 @@ RUN grep security /etc/apt/sources.list > /etc/apt/security.sources.list && \
 ##
 FROM sentinel-base
 
+# If you copy from /usr/share/sentinel to /usr/share/sentinel the permissions are not preserved
+# We would have 755 for sentinel:root instead of 775 and prevents writing lock files in /usr/share/sentinel
+COPY --chown=10001:0 --from=sentinel-tarball /opt/usr-share /usr/share
+
 SHELL ["/bin/bash", "-c"]
 
 # Create Sentinel user with a specific group ID
 RUN groupadd -g 10001 sentinel && \
     useradd \
       --system \
       --uid 10001 \
-      --gid 10001 \
+      --gid 0 \
```

Comment:
Was this meant to be a 0 gid? It's still 10001 in Horizon.

Reply:
Oops good catch, I'll fix it in post. ;)